main_linux.go

```diff
@@ -173,7 +176,8 @@ func main() {
 				Event     string
 				Shell     string
 				Threshold int
-			}{event, shell, current}
+				Path      string
+			}{event, shell, current, first}
```

Comment:
We can reduce the size of this struct by placing the `Threshold` field last.

Reply:
How does that change the size?

Reply:
I mean it doesn't affect the size per se, just [how the garbage collector does the mark phase](https://cs.opensource.google/go/x/tools/+/refs/tags/v0.13.0:go/analysis/passes/fieldalignment/fieldalignment.go;drc=e5f7ae13a79f624b7414958de3f8c64693adfe8c;l=29).